README.md

```diff
@@ -8,47 +8,31 @@ Rack and Faraday integration middlewares for Zipkin tracing.
-as a hash argument to the Rack plugin.
-
-    require 'zipkin-tracer'
-    use ZipkinTracer::RackHandler, config # config is optional
-
-where Rails.config.zipkin_tracer or config is a hash that can contain the following keys:
-
- * `:service_name` (REQUIRED) - the name of the service being traced. There are two ways to configure this
-   value. Either write the service name in the config file or set the "DOMAIN"
-   environment variable (e.g. 'test-service.example.com' or 'test-service'). The
-   environment variable takes precedence over the config file value.
- * `:service_port` (REQUIRED) - the port of the service being traced (e.g. 80 or 443)
- * `:scribe_server` (default from scribe gem) - the address of the scribe server where traces are delivered
- * `:scribe_max_buffer` (default: 10) - the number of annotations stored until automatic flush
-       (note that annotations are also flushed when the request is complete)
- * `:sample_rate` (default: 0.1) - the ratio of requests to sample, from 0 to 1
- * `:annotate_plugin` - plugin function which recieves Rack env, and
-   response status, headers, and body; and can record annotations
- * `:filter_plugin` - plugin function which recieves Rack env and will skip tracing if it returns false
- * `:whitelist_plugin` - plugin function which recieves Rack env and will force sampling if it returns true
- * `:zookeeper` - plugin function which uses zookeeper and kafka instead of scribe as the transport
- * `:logger` - A logger class following the standard's library Logger interface (Log4r, Rails.logger, etc).
- * `:json_api_host` - hostname with protocol of a zipkin api instance (e.g. `https://zipkin.example.com`) to use JSON over HTTP instead of Scribe or Kafka
- * `:traces_buffer` (default: 100) - the number of annotations stored until automatic flush
-   (note that annotations are also flushed when the request is complete)
+Options can be provided via Rails.config for a Rails 3+ app, or can be passed as a hash argument to the Rack plugin.
 
+```ruby
+require 'zipkin-tracer'
+use ZipkinTracer::RackHandler, config # config is optional
+```
 
- If the configuration does not provide either a JSON, Scribe or Zookeeper server then the middlewares will not
- attempt to send traces although they will still generate proper IDs and pass them to other services.
- Thus, if you only want to generate IDs for instance for logging and do not intent to integrate with Zipkin,
- you can still use this gem. Just do not specify any server :)
+where `Rails.config.zipkin_tracer` or `config` is a hash that can contain the following keys:
 
+* `:service_name` **REQUIRED** - the name of the service being traced. There are two ways to configure this value. Either write the service name in the config file or set the "DOMAIN" environment variable (e.g. 'test-service.example.com' or 'test-service'). The environment variable takes precedence over the config file value.
```

Comment:
If only making tables in MD wasn't such a PITA and I wasn't so lazy on a Monday morning... :sweat_smile: